package.json

```diff
@@ -37,6 +37,7 @@
   "main": "./out/src/extension.js",
   "activationEvents": [
     "onView:truffle-vscode.truffle",
+    "onView:truffle-vscode.explorer-view",
```

Comment:
As discussed, for the explorer view if we could just show either just contents of the `contracts` directory (or ideally any directories that contain contracts if the project is using a monorepo, although appreciate that's way less trivial) that would awesome. Also for the right-click context menus, which look like they have to be manually wired up, the current commands that we display for `.sol`'s, e.g. `Build Contracts`, `Deploy Contracts`, etc

Reply:
contract dir showing, still need to work on the build deploy button integration/context menus